Formula/antidote.rb

```diff
@@ -0,0 +1,34 @@
+class Antidote < Formula
+  desc "Plugin manager for zsh, inspired by antigen and antibody"
+  homepage "https://getantidote.github.io/"
+  url "https://github.com/mattmc3/antidote/archive/refs/tags/v1.6.4.tar.gz"
+  sha256 "336ea89aab33d07d5d2eaa14af2f015d7da657837f64d3e76fa5c54645210950"
+  license "MIT"
+
+  uses_from_macos "zsh"
+
+  def install
+    pkgshare.install "antidote.zsh"
+    pkgshare.install "functions"
+    rm_f Dir["man/*.md"]
+    man.install Dir["man/*"]
```

Comment:
```suggestion
    rm_f Pathname.glob("man/*.md")
    share.install "man"
```
You're not operating over directories, so let's not use `Dir`.